plasmapy/classes/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .plasma import Plasma
+from .plasma import Plasma3D, PlasmaBlob
```

Comment:
`docs/plasma/index.rst` will probably need to change as a result of this.

Reply:
Updated. Please verify that I did this correctly.

Reply:
:+1: